src/garage/sampler/vec_worker.py

```diff
@@ -147,14 +147,22 @@ def start_rollout(self):
 
     def _gather_rollout(self, rollout_number, last_observation):
         assert 0 < self._path_lengths[rollout_number] <= self._max_path_length
+        env_infos = self._env_infos[rollout_number]
+        self._env_infos[rollout_number] = collections.defaultdict(list)
```

Comment:
In a separate pr, you and I actually make the same change, but I have some additional changes on top of what you have made. We should merge our PRs together.

Reply:
Should I close this PR and open a new PR to merge your branch?